main.go

```diff
@@ -33,30 +38,54 @@ func run(args []string, stdout io.Writer) error {
 	if len(args) == 0 {
 		return errors.New("no arguments")
 	}
+
 	flags := flag.NewFlagSet(args[0], flag.ExitOnError)
-	flag.BoolVar(&debug, "debug", false, "sets log level to debug and console pretty output")
+
+	debug := flag.Bool("debug", false, "sets log level to debug and console pretty output")
+
+	// (&debug,
+	// 	"debug",
+	// 	false,
+	// 	"sets log level to debug and console pretty output")
 
 	if err := ff.Parse(flags, args); // ff.WithEnvVarNoPrefix(),
 	// ff.WithConfigFileFlag("config"),
 	// ff.WithConfigFileParser(fftoml.Parser),
 	err != nil {
 		return err
 	}
-	// Logger.Configure(Logger.Config{
-	// 	ConsoleLoggingEnabled: true,
-	// })
-	// l.Log.Info().Msg("func run() completed")
-	// Logger.Info().Msg("func run() completed")
+
+	LogLevel := "info"
+	if *debug {
+		LogLevel = "debug"
+	}
+
+	c := logger.Config{
+		Enable:                true,
+		ConsoleLoggingEnabled: true,
+		EncodeLogsAsJson:      false,
+		FileLoggingEnabled:    false,
+		Directory:             "",
+		Filename:              "",
+		MaxSize:               MaxSize,
+		MaxBackups:            MaxBackups,
+		MaxAge:                MaxAge,
+		Level:                 LogLevel,
+	}
+
+	_ = logger.InitLogger(c)
```

Comment:
No point in using the return in this current approach as I'm providing a package level variable in `logger` that already is exported and available to use.